string model_name
int32    order
uint32[] epochs
---
uint32[] epochs
float64[] predictions